assets/messages.yml

```diff
@@ -856,6 +856,21 @@ messages:
         {quote}Problems related to modifying internal NBT tags during the lifetime of an entity are not considered bugs.
         -- [~searge]'s comment [here|https://bugs.mojang.com/browse/MC-66943?focusedCommentId=188032&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-188032] in MC-66943.{quote}
 
+        %quick_links%
+      fillname: []
+  modified-world-crash:
+    - project:
+        - mc
+        - mcl
+        - mctest
+      name: Modified world crash
+      shortcut: mworld
+      message: |-
+        The attached crash report indicates that you had opened your world with a 'modded' game in the past.
+        We only accept reports about the unmodified ('vanilla') game here on the bug tracker. To rule out that the modified game is responsible for the crash,
+        please try reproducing this issue with a newly created world or a world which had not been opened with a modified game in the past. If you experience
+        the crash there as well please attach that crash report too. Otherwise please report this issue to the author of the modification instead.
```

Comment:
```suggestion
        the crash there as well, please attach that crash report too. Otherwise, please report this issue to the author of the modification instead.
```